tutorials/10_DebuggingREMIND.md

```diff
@@ -0,0 +1,183 @@
+Debugging REMIND
+================
+Collaborative effort, coordinated by Oliver Richters, 31 March, 2022
+
+When running and changing REMIND, a multitude of errors may occur. In such a collaborative effort, they need not have been created by you and the REMIND team will help you solve them. So an important part of debugging is to find out whether something you changed created the error, or whether you were unfortunate enough to find errors you are not responsible for.
```

Comment:
could we formulate this a bit more less promising that the REMIND team will help with debugging? I am will add another general disclaimer that we can only provide limited support. Maybe just delete " and the REMIND team will help you solve them" and instead of "or whether you were unfortunate enough to find errors you are not responsible for." state "or whether there is a general problem".  Please, also try to avoid the impression that REMIND has a tons of bugs :-)

Reply:
maybe add a statement that releases are stable versions of the model and the develop branch is permanently developed and therefore also might contain bugs or inconsistencies

Reply:
that is what I did

Reply:
please delete " and the REMIND team will help you solve them" and maybe change "or whether you were unfortunate enough to find errors you are not responsible for." to "or whether there is a general problem"